CONTRIBUTING.md

```diff
@@ -1,70 +1,103 @@
-## Contributing
+# Contributing
 
 | Component | Build Status |
 | --------- | ------------ |
 | Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
 | Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
 
-### Prerequisites:
+# Prerequisites:
   Install VS 2017 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
```

Comment:
At least for the client libraries this is going to need to be VS 2019 or higher.